docs/tutorials/excel-power-automate-trigger.md

```diff
@@ -1,21 +1,23 @@
 ---
 title: 'Pass data to scripts in an automatically-run Power Automate flow'
 description: 'A tutorial about running Office Scripts for Excel on the web through Power Automate when mail is received and passing flow data to the script.'
-ms.date: 07/14/2020
+ms.date: 07/20/2020
 localization_priority: Priority
 ---
 
 # Pass data to scripts in an automatically-run Power Automate flow (preview)
 
 This tutorial teaches you how to use an Office Script for Excel on the web with an automated [Power Automate](https://flow.microsoft.com) workflow. Your script will automatically run each time you receive an email, recording information from the email in an Excel workbook.
 
+Completing this tutorial help you understand how to run scripts based on outside information. You'll see how flows can be triggered automatically as well as how data is passed between flow steps, especially for input into an Office Script.
+
+> [!TIP]
+> If you are new to Office Scripts, we recommend starting with the [Record, edit, and create Office Scripts in Excel on the web](excel-tutorial.md) tutorial. If you are new to Power Automate, we recommend starting with the [Call scripts from a manual Power Automate flow](excel-power-automate-manual.md) tutorial.
+
 ## Prerequisites
 
 [!INCLUDE [Tutorial prerequisites](../includes/power-automate-tutorial-prerequisites.md)]
```

Comment:
I'm flexible here, but my logic is we can level-set in the first tutorial. If every other tutorial says "take the first tutorial" and the first tutorial says "know some JS/TS", then we can assume people have seen that message in the most important spot but don't have to repeatedly see it. A lot of this PR is based on feedback that there's a lot of stuff before the tutorial that distracts from the important bits.

Reply:
Is that feedback from customers? Just curious. I know people are opinionated on this one but we get so much traffic through search that it seems like it would be helpful to indicate it here